docs/04-guides/playback-a-video-stream.mdx

```diff
@@ -6,16 +6,166 @@ description: Playback video and livestreams
+        "playbackId": "7d048tbcfrv6gvzs",
+        // highlight-end
+        "userId": "80dc8f6e-69d5-401f-bbd7-bfc09a2a5320",
+        "createdAt": 1659715086322,
+        "status": {
+            "phase": "waiting",
+            "updatedAt": 1659715086322
+        },
+        "name": "Example name"
+    },
+    "task": {
+        "id": "d3a2ab1b-8dd1-450d-ac8e-498fd9d91865"
+    }
+}
+```
+
+### Installation
+
+Once you have the playback id, the next step is to install the library. You can
+do so by running the following command:
+
+<Tabs>
+  <TabItem value="npm" label="npm">
+    <CodeBlock language="bash">npm install @livepeer/react</CodeBlock>
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+    <CodeBlock language="bash">yarn add @livepeer/react</CodeBlock>
+  </TabItem>
+</Tabs>
+
+### Usage
+
+The Player component provides an easy way to display video or audio. You can
+also customize the player's theme and controls.
+
+```jsx
+import { Player } from "@livepeer/react";
+
+export const SimplePlayer = () => {
+  return (
+    <Player
+      title="Agent 327: Operation Barbershop"
+      playbackId="6d7el73r1y12chxr"
+      showPipButton
+    />
+  );
+};
+```
+
+<Player />
+
+## Using your own player
+
+Using livepeer.js is the easiest and recommended way to playback a video or a
+live stream. However, if you want to use your own player, you can do so by
+following the instructions below.
+
+:::caution
+
+Please note that to playback live streams inside your application you'll need to
+use a video player component that supports the HLS.
+
+:::
+
+### Create a playback URL
+
+To playback a live stream in other players, you'll need to create an HLS URL.
+HLS is a protocol that allows you to stream video and audio content over HTTP.
+Most of the video you watch on the web is delivered using HLS. Livepeer also
+uses HLS to deliver video and audio content.
+
+You can create a HLS url by using the below format and replacing the
+`PLAYBACK_ID` with your stream's playback id.
+
+```txt
+https://livepeercdn.com/hls/{PLAYBACK_ID}/index.m3u8
+```
+
+However, if you want to playback a video that is already uploaded to Livepeer,
+you can use the below format and replace the `PLAYBACK_ID` with your video's
+playback id.
+
+```txt
+https://livepeercdn.com/asset/{PLAYBACK_ID}/video
+```
+
+### Use the playback URL in a player
+
+You can use the playback URL is any video player that supports HLS. Here is a
+list of popular players that supports HLS:
+
+- [Video.js](https://videojs.com/)
+- [Plyr.io](https://plyr.io/)
+- [HLS.js](https://github.com/video-dev/hls.js)
+- [JW Player](https://www.jwplayer.com/html5-video-player/)
+- [Bitmovin Player](https://bitmovin.com/video-player/)
+
+Here is an example of how to use the playback URL in a `video` tag.
+
+```jsx
+import React from "react";
+
+export default function Player() {
+  return (
+    <div>
+      <video
+        controls
+        src="https://livepeercdn.com/asset/6d7el73r1y12chxr/video"
```

Comment:
Hmm does this work? I think it will only work on Apple devices by default. We might want to give a working example here, maybe with video.js or hls.js.